src/server/game/Server/Protocol/Opcodes.h

```diff
@@ -1042,7 +1042,7 @@ enum OpcodeServer : uint32
     SMSG_CORPSE_TRANSPORT_QUERY                       = 0x0B6C,
     SMSG_CREATE_SHIPMENT_RESPONSE                     = 0xBADD,
     SMSG_CREATURE_QUERY_RESPONSE                      = 0x0DC2,
-    SMSG_CRITERIA_DELETED                             = 0x0B2A,
+    SMSG_CRITERIA_DELETED                             = 0xBADD,
```

Comment:
ups sorry, i change it after the work :) but CMSG_TAXI_QUERY_AVAILABLE_NODES is correct edit.